cdap-distributions/src/hdinsight/mainTemplate.json

```diff
@@ -50,18 +50,18 @@
       },
       "installScriptActions": [{
         "name": "[concat('cdap-pageblob-configure-v0','-' ,uniquestring(variables('applicationName')))]",
-        "uri": "https://raw.githubusercontent.com/caskdata/cdap/release/3.5/cdap-distributions/src/hdinsight/pageblob-configure.sh",
+        "uri": "http://downloads.cask.co/hdinsight/pageblob-configure-3.5.1.sh",
```

Comment:
This is incorrect. The script is created and published as part of the release. These files don't exist and won't exist until the release is published. That being said, we *will* need to change them before the release is made, so this is the end-goal. It's just that currently, these scripts won't work, as is.